### PR TITLE
Initial batch create form

### LINKED ISSUE #4: Create Form for Batches

### LINKED ISSUE #21: Add a listing of all of a user's batches to the screen.
Super rough paper wireframes:

https://docs.google.com/drawings/d/12YhQ6IO9whvBAGErG_ygl0O3bheNVNppzsS4hhovchM/edit

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #4 
Closes #21

Screen Shot 2021-06-04 at 10 30 48 AM

Screen Shot 2021-06-04 at 12 41 24 PM

### CODE REVIEW

File: app/views/batches/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+      </div>
+      <%= f.button :submit, class: 'btn-primary' %>
+    <% end %>
+  </div>
+</div>
+
+<h2>Unsynchronized</h2>
+<div id="unsynchronized-batches" class="bg-light col m-3 p-3">
+  <% (current_user&.batches&.order(created_at: :desc) || []).each do |batch| %>
```
Comment (eliotjordan) [inline app/views/batches/index.html.erb:38]: question: How are synchronized batches filtered out?

**Review by eliotjordan — COMMENTED** [review]
Looks good. I do have a couple of questions.

File: app/views/batches/index.html.erb (hunk 1 above)
Comment (tpendragon) [reply app/views/batches/index.html.erb:38]: It isn't right now, I figured we could handle that when synchronizing was a thing.

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -13,10 +13,13 @@ import "lux-design-system/dist/system/tokens/tokens.scss";
 import "channels"
 import "bootstrap"
 import "../stylesheets/application"
+import $ from 'jquery';
+import dt from 'datatables.net-bs4';
 
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+console.log(dt.version)
```
[the remark this answers is not in the document]
Comment (tpendragon) [reply app/javascript/packs/application.js]: Nope, removed.

**Review by eliotjordan — APPROVED** [review]

**Pull request merged**